pkg/controller/dataimportcron-controller_test.go

```diff
@@ -340,6 +343,12 @@ var _ = Describe("All DataImportCron Tests", func() {
 			Expect(*dv.Spec.Source.Registry.URL).To(Equal(testRegistryURL + "@" + testDigest))
 			Expect(dv.Annotations[cc.AnnImmediateBinding]).To(Equal("true"))
 
+			if cron.Spec.Schedule == emptySchedule {
+				exists, err := reconciler.cronJobExistsAndUpdated(context.TODO(), cron)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(exists).To(BeFalse())
+			}
+
```

Comment:
We should probably also ensure the `batchv1.Job{}` doesn't get created

Reply:
Thanks. fixed both comments